central/imageintegration/datastore/internal/search/searcher_impl.go

```diff
@@ -0,0 +1,42 @@
+package search
+
+import (
+	"context"
+
+	imageIntegrationMapping "github.com/stackrox/rox/central/imageintegration/mappings"
+	"github.com/stackrox/rox/central/imageintegration/store"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/search/blevesearch"
+	"github.com/stackrox/rox/pkg/search/paginated"
+	"github.com/stackrox/rox/pkg/search/sortfields"
+)
+
+var (
+	defaultSortOption = &v1.QuerySortOption{
+		Field:    search.DeploymentID.String(),
```

Comment:
image integrations do not have a `DeploymentID`. Perhaps you can just do `ClusterID` instead